Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -479,20 +480,47 @@ ILocalizationProvider localizationProvider
                 );
             }
 
-            if (command.CommandMethod.GetParameters().Length > 0)
+            var parameters = command.CommandMethod.GetParameters();
+            if (parameters.Length > 0)
             {
-                return new UnsupportedFeatureError
-                (
-                    "Context menu commands may not have parameters.",
-                    command
-                );
+                var expectedParameter = commandType switch
+                {
+                    ApplicationCommandType.Message => "message",
+                    ApplicationCommandType.User => "user",
```

Comment:
In this context, what if the user clicks on a message to bring up the user profile to use the command, would it be possible to obtain that message then?

Reply:
No, it would not be possible as in the user command context the message is not available.
It chooses the channel, the command response is sent to, depending on the open channel.

Reply:
Except when you reply in the command it actually somehow gets the message from where you right clicked them and then reply to it.

Reply:
What do you mean by reply?

Reply:
When you use ``IDiscordRestInteractionAPI.CreateFollowupMessageAsync`` in the user context command.

Reply:
It does not reference at all the message you used to bring up the user profile to use the command.
`IDiscordRestInteractionAPI.CreateFollowupMessageAsync` is just a reply to the interaction itself.

Reply:
I mean technically what you are referring to is actually a message command.
You can get the user of the message from the Command context itself.

Reply:
Just now changed the command from ``User`` to Message, waiting on when this can be merged now 👌.